packages/lesswrong/components/community/Community.tsx

```diff
@@ -0,0 +1,325 @@
+  const [mapsLoaded, googleMaps] = useGoogleMaps("CommunityHome")
+  const [geocodeError, setGeocodeError] = useState(false)
+  const saveReverseGeocodedLocation = async ({lat, lng, known}) => {
+    if (
+      mapsLoaded &&     // we need Google Maps to be loaded before we can call the Geocoder
+      !geocodeError &&  // if we've ever gotten a geocoding error, don't try again
+      known             // skip if we've received the default location
+    ) {
+      try {
+        // get a list of matching Google locations for the current lat/lng (reverse geocoding)
+        const geocoder = new googleMaps.Geocoder();
+        const geocodingResponse = await geocoder.geocode({
+          location: {lat, lng}
+        });
+        const results = geocodingResponse?.results;
+        
+        if (results?.length) {
+          const location = pickBestReverseGeocodingResult(results)
+          saveUserLocation({lat, lng, gmaps: location})
+        } else {
+          // update the page state to indicate that we're not waiting on a location name
+          userLocation.setLocationData({...userLocation, label: null})
+        }
+      } catch (e) {
+        setGeocodeError(true)
+        // eslint-disable-next-line no-console
+        console.error(e?.message)
+        // update the page state to indicate that we're not waiting on a location name
+        userLocation.setLocationData({...userLocation, label: null})
+      }
+    }
+  }
+
+  // on page load, try to get the user's location from:
+  // 1. (logged in user) user settings
+  // 2. (logged out user) browser's local storage
+  // 3. failing those, browser's geolocation API (which we then try to save in #1 or #2)
+  const userLocation = useUserLocation(currentUser)
+  
+  useEffect(() => {
+    // if we've gotten a location from the browser's geolocation API, save it
+    if (!userLocation.loading && userLocation.known && userLocation.label === '') {
+      void saveReverseGeocodedLocation(userLocation)
+    }
+    //eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [userLocation])
+
+  const openEventNotificationsForm = () => {
+    openDialog({
+      componentName: currentUser ? "EventNotificationsDialog" : "LoginPopup",
+    });
+  }
+  
+  const { CommunityBanner, LocalGroups, OnlineGroups } = Components
+  
+  const handleChangeTab = (e, value) => {
+    setTab(value)
+    history.replace({...location, hash: `#${value}`})
+  }
+
+  return (
+    <AnalyticsContext pageContext="Community">
+      
+      <div className={classes.section}>
+        <div className={classes.sectionHeadingRow}>
+          <h1 className={classes.sectionHeading}>
+            Community
+          </h1>
+          <div className={classes.sectionDescription}>
+            Effective altruism is a global community with thousands of members. Reach out to learn how you can have the most impact.
+          </div>
+        </div>
+      </div>
+        
+      {/* <CommunityBanner /> */}
```

Comment:
I made a banner similar to the one Lotte designed [here](https://framer.com/share/Prototype-CEA--jXrr1x7ljK9p4nBODAQ8/Chjgs_sno?editor=1), but since I didn't have a good quote I'm excluding it for now (the placeholder one in the component is made up). If I get a good quote I'll finish building the banner and add it back in.

Reply:
Ok I asked Vaidehi if I could use a quote from [this post](https://forum.effectivealtruism.org/posts/kE3FRC5gq9QxMrn3w/what-drew-me-to-ea-reflections-on-ea-as-relief-growth-and) and they said it was ok :)

<img width="1206" alt="Screen Shot 2022-02-24 at 5 45 59 PM" src="https://user-images.githubusercontent.com/9057804/155620345-a03e968a-e88c-446c-9556-8ea58ddca9f6.png">